src/Services/Utils/UtilsValidator.js

```diff
@@ -1,4 +1,5 @@
 const { UtilsValidationError } = require('./UtilsErrors');
+const referencedatatypes = require('../../reference-data-types');
```

Comment:
nice validator! let's make it more readable by using snakeCase?

Reply:
Did you mean camelCase ? I made that way